.github/workflows/deploy.yml

```diff
@@ -8,37 +8,119 @@ on:
     branches: [ master ]
 
 jobs:
-  deploy:
-    name: Deploy Demo App
+  deploy_chime_alpha_demo:
+    name: Alpha - Deploy Meeting Demo And Meeting Readiness Checker
     runs-on: ubuntu-latest
     env:
-        GAMMA_AWS_ACCESS_KEY_ID: ${{secrets.GAMMA_AWS_ACCESS_KEY}}
-        GAMMA_AWS_SECRET_ACCESS_KEY: ${{secrets.GAMMA_AWS_SECRET_ACCESS_KEY}}
-        BETA_AWS_ACCESS_KEY_ID: ${{secrets.BETA_AWS_ACCESS_KEY}}
-        BETA_AWS_SECRET_ACCESS_KEY: ${{secrets.BETA_AWS_SECRET_ACCESS_KEY}}
         AWS_DEFAULT_REGION: us-east-1
         AWS_DEFAULT_OUTPUT: text
+        STAGE: Alpha
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install SAM CLI
+        run: script/github-action-awscli-installation
+      - name: Run Deployment Script
+        run: script/deploy-canary-demo
+  deploy_chime_beta_demo:
+    name: Beta - Deploy the Serverless Meeting and Meeting Readiness Checker Demos that talks to Chime Client
+    runs-on: ubuntu-latest
+    env:
+        AWS_DEFAULT_REGION: us-east-1
+        AWS_DEFAULT_OUTPUT: text
+        STAGE: ChimeBeta
```

Comment:
Shouldn't this be just `Beta` to be inconsistent with `Alpha` like above? Same for below stage?

Reply:
Alpha does not have two clients that is why i just put alpha and beta and gamma has two clients "Chime" and "ChimeMeetings"